api/db/migrate/20181130095058_add_name_to_tournaments.rb

```diff
@@ -0,0 +1,5 @@
+class AddNameToTournaments < ActiveRecord::Migration[5.2]
+  def change
+    add_column :tournaments, :name, :string, default: ''
```

Comment:
Please generate name for all existing tournaments and remove default empty string. Also please add null: false